src/plugins/telemetry/server/routes/telemetry_opt_in_stats.ts

```diff
@@ -35,8 +34,10 @@ export async function sendTelemetryOptInStatus(
     channelName: 'optInStatus',
   });
 
-  const optInStatusPayload: UnencryptedTelemetryPayload =
```

Comment:
I removed `UnencryptedTelemetryPayload` because it's not entirely true... It could also return `EncryptedTelemetryPayload` (actually, attending to the usage, it's the most likely output) 😇